src/main.ts

```diff
@@ -62,6 +75,12 @@ const command: Command = {
 			default: false
 		});
 
+		options('i', {
```

Comment:
I disagree...  What if you want to change to the browsers you are validating against in SauceLabs/BrowserStack/TestingBot, or you want to test against Safari or Firefox using SeleniumTunnel?  You have to totally go it alone now?